src/index.html

```diff
@@ -4,9 +4,401 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#" class="logo-link">
```

Comment:
`logo-link` also an element of `catalog` block 

`<a href="#" class="header__logo-link">`

![Screenshot 2022-07-19 at 07 13 57](https://user-images.githubusercontent.com/17098736/179663215-15f678b6-8bbd-45c5-bc6a-5d306674dea3.png)